index.html

```diff
@@ -1,203 +1,233 @@
+            <li><a href="https://www.linkedin.com/in/muthomi-titus-295024181/"><img src="./img/linkedin.png" alt=""></a>
+            </li>
+            <li><a href="https://medium.com/@muthomititus"><img src="./img/medium.png" alt=""></a></li>
+            <li><a href="https://twitter.com/titus_muthomi"><img src="./img/twitter.png" alt=""></a></li>
+            <li><a href="#"><img src="./img//vector.svg" alt=""></a></li>
+          </ul>
+
+          <button class="buttons" type="submit">
+            Get My Resume
+          </button>
         </div>
-        <div class="same">
-          <img src="./img//css.png" alt="">
-          <span class="lists">CSS</span>
+
+        <div class="langandframe">
+          <div class="language">
+            <span class="languages">Languages</span> <span class="material-icons">expand_more</span>
+          </div>
+
+          <div class="language-list">
+            <div class="same">
+              <img src="./img/js.png" alt="">
+              <span class="lists">Javascript</span>
+            </div>
+            <div class="same">
+              <img src="./img//html.png" alt="">
+              <span class="lists">HTML</span>
+            </div>
+            <div class="same">
+              <img src="./img//css.png" alt="">
+              <span class="lists">CSS</span>
+            </div>
+          </div>
+
+          <div class="language">
+            <span class="languages">Frameworks</span> <span class="material-icons">chevron_right</span>
+          </div>
+          <div class="language">
+            <span class="languages">Skills</span> <span class="material-icons">chevron_right</span>
+          </div>
+
         </div>
-      </div>
-
-      <div class="language">
-        <span class="languages">Frameworks</span> <span class="material-icons">chevron_right</span>
-      </div>
-      <div class="language">
-        <span class="languages">Skills</span> <span class="material-icons">chevron_right</span>
-      </div>
-      
-    </div>
-
-  </section>
-
-  <section class="contacts">
-    <h2 class="contact-me">Contact Me</h2> 
-    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
-    <form action="https://formspree.io/f/meqvlald" method="POST">
-      <label >
-        <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>
-      </label>
-      <label class="mail-input">
-        <input type="email" id="email" placeholder="Email" name="_replyto" required>
-      </label>
-      <label>
-      <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write Your message here" required></textarea>
-      </label>
-      <button type="submit">Get in Touch</button>
-    </form>
-  </section>
-</body>
+
+      </section>
```

Comment:
- [ ] Below 1000px width about section's align is broken. ıt should be center like language section.

You should fix this.
![image](https://user-images.githubusercontent.com/38635158/125699620-f44efb86-d87d-43c7-9064-69a22fb86e1c.png)